low-level/files/opt/seagate/sspl/bin/sspl_reset.py

```diff
@@ -0,0 +1,174 @@
+import sys
+import errno
+import argparse
+import inspect
+import traceback
+import os
+import shutil
+import subprocess
+# Add the top level directories
+sys.path.insert(0, '/opt/seagate/cortx/sspl/low-level')
+
+from framework.base.sspl_constants import PRODUCT_FAMILY, file_store_config_path, enabled_products
+
+SSPL_CONFIGURED=f"/var/{PRODUCT_FAMILY}/sspl/sspl-configured"
+
+if os.path.exists(file_store_config_path):
+    with open(file_store_config_path, "r") as SSPL_CONF:
+        for line in SSPL_CONF:
+            if 'data_path' in line:
+                SSPL_DATA_DIR=line.split("=")[1]
+            elif 'log_file_path' in line:
+                IEM_FILE_PATH=line.split("=")[1]
+else:
+    SSPL_DATA_DIR=f"/var/{PRODUCT_FAMILY}/sspl/data"
+    IEM_FILE_PATH=f"/var/{PRODUCT_FAMILY}/sspl/data/iem/iem_messages"
+
+class Cmd:
+
+    def __init__(self, args: dict):
+        self._args = args.args
+        
+
+    @property
+    def args(self) -> str:
+        return self._args
+
+    @staticmethod
+    def usage(prog: str):
+        """Print usage instructions"""
+
+        sys.stderr.write(
+            f"usage: {prog} [hard [-p <LDR_R1>]|soft]\n"
```

Comment:
LDR_R2.
Please check the latest main branch. Mariyappan has made some changes through #264.

Reply:
Done.